eZ/Bundle/EzPublishCoreBundle/Command/UpdateTimestampsToUTCCommand.php

```diff
@@ -151,17 +153,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 sprintf('The selected mode is not supported. Use one of the following modes: %s', implode(', ', array_keys(self::MODES)))
             );
 
-            return;
+            return 0;
```

Comment:
I think we should start using non-zero for error states. That's the whole point of exit status codes :)

Reply:
I agree but was concerned if the case that is handled by the application qualifies as error exit code. :) It basically ends up without any issue, mostly is either misusage of command options or no records found. WDYT @alongosz?

Reply:
In my own apps, I gravitate more and more to 'proper' usage of the unix facilities for command-line scripts. This is because it makes it easy to then wrap calls to those scripts in shell scripts and properly handle all cases.
This means:
- all error messages, even if not fatal, are echoed to stderr, not stdout
- return non-0 exit code whenever there was at least one malfunction within the script, even if not fatal (eg. a notification script fails to send 1 mail, or to load 1 content)

Reply:
ps: also, same comment as below: for invalid args/options, I'd say simplest thing to do is to throw. I am sure there are appropriate exceptions...